GameAssistant/Services/DieRollerService.cs

```diff
@@ -1,6 +1,9 @@
-﻿namespace GameAssistant.Services
+﻿using GameAssistant.Models;
+
+namespace GameAssistant.Services
 {
-    class DieRollerService
+    public class DieRollerService<T>
```

Comment:
is this used for anything? can it be removed?

Reply:
Will probably remove later